mkt/webapps/tests/test_serializers.py

```diff
@@ -203,6 +204,14 @@ def test_release_notes(self):
         res = self.serialize(self.app)
         eq_(res['release_notes'], unicode(version.releasenotes))
 
+    def test_size(self):
+        date = datetime.datetime.now() + datetime.timedelta(days=10)
+        version = self.app.current_version
+        f = version.files.create(size=12345)
```

Comment:
There should already be a file created with the `app_factory`. Instead of creating another one why not query the existing one?